provider/inspec.rb

```diff
@@ -265,7 +265,9 @@ def resource_name(object, product)
     # Recursively calls itself on any arrays or nested objects within this property, indenting
     # further for each call
     def markdown_format(property, indent = 1)
-      prop_description = "`#{property.out_name}`: #{property.description.split("\n").join(' ')}"
+      beta_description = property.min_version.name == 'beta' ? '(Beta only) ' : ''
```

Comment:
Is min_version always going to be not nil?

Reply:
Correct: https://github.com/GoogleCloudPlatform/magic-modules/blob/1307550a3ab01b20c2f8410d51c191e190d33d30/api/type.rb#L249